### PR TITLE
update basic features and experimental features

### PR DESCRIPTION
**Pull request by ran-huang**
### What is changed, added or deleted? (Required)

Update basic features and experimental features for v6.1.0. The changes are listed as follows:

- List 分区: experimental -> GA
- List COLUMNS 分区: experimental -> GA
- 动态裁剪: experimental -> GA
- Global Kill: experimental -> GA
- SET CONFIG（动态修改配置项）: experimental -> GA
- Raft Engine： experimental -> GA
- 在线有损恢复： experimental -> GA
- 使用 Bucket 增加并发：experimental
- TiKV API v2: experimental
- 限制 ANALYZE 的内存使用量： experimental
- ALTER TABLE ... COMPACT: experimental
- User level lock: GA
- Batch: GA
- Statistics for dynamic pruning: Experimental -> GA

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v6.0 (TiDB 6.0 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### CODE REVIEW

**Review by moyun — COMMENTED** [review]
LGTM

**Comment by songrijie** [comment]
LGTM

**Review by hfxsd — APPROVED** [review]
LGTM

File: experimental-features.md [hunk 1]
```diff
@@ -49,6 +49,8 @@ aliases: ['/docs-cn/dev/experimental-features-4.0/','/zh/tidb/dev/experimental-f
 + [关闭 Titan 功能](/storage-engine/titan-configuration.md#关闭-titan实验功能)。（v4.0 实验特性）
 + [Titan Level Merge 功能](/storage-engine/titan-configuration.md#level-merge实验功能)。（v4.0 实验特性）
 + TiFlash 支持将存储引擎的新数据分布在多个硬盘上，分摊 I/O 压力。（v4.0 实验特性）
++ 将 Region 划分为更小的区间 bucket，并且[以 bucket 作为并发查询单位](/tune-region-performance.md#使用-bucket-增加并发)，以提高扫描数据的并发度。（v6.1.0 实验特性）
```
Comment (hfxsd) [inline experimental-features.md:52]: 这个链接的文档里要加上实验特性的说明

**Review by TomShawn — APPROVED** [review]

File: experimental-features.md (hunk 1 above)
Comment (easonn7) [reply experimental-features.md:52]: Sprint 8 追加特性如下：
数据类型，函数和操作符 ：｜用户级别锁｜ 6.1 GA ｜ /functions-and-operators/locking-functions.md｜之前 N/A
SQL 语句： ｜batch [on column] limit integer delete｜ 6.1 GA ｜/sql-statements/sql-statement-batch.md ｜之前 N/A
统计信息：｜动态剪裁 ｜ 6.1 GA ｜/partitioned-table.md ｜ 之前版本是 实验特性，具体版本信息待确认

Comment (ran-huang) [reply experimental-features.md:52]: Done. 
动态剪裁从文档里看是 v5.1 (https://docs.pingcap.com/zh/tidb/v5.1/partitioned-table#%E5%8A%A8%E6%80%81%E8%A3%81%E5%89%AA%E6%A8%A1%E5%BC%8F) 开始实验特性，v5.0 及以前 N/A。

File: basic-features.md [hunk 2]
```diff
@@ -119,61 +121,62 @@ aliases: ['/docs-cn/dev/basic-features/']
...
-| [统计反馈](/statistics.md#自动更新)                       | 已废弃 |  已废弃  | 实验特性 | 实验特性 | 实验特性 | 实验特性 | 实验特性 |
-| [快速分析](/system-variables.md#tidb_enable_fast_analyze) | 实验特性 | 实验特性 | 实验特性 | 实验特性 | 实验特性 | 实验特性 | 实验特性 |
+| 统计信息                                                  |   6.1    |   6.0    |   5.4    |   5.3    |   5.2    |   5.1    |   5.0    | 4.0      |
+| --------------------------------------------------------- | :------: | :------: | :------: | :------: | :------: | :------: | :------: | -------- |
+| [CM-Sketch](/statistics.md)                               | 已废弃 | 已废弃 |  已废弃  |  已废弃  |  已废弃  |  已废弃  |  已废弃  | Y        |
+| [直方图](/statistics.md)                                  |   Y   |   Y   |    Y     |    Y     |    Y     |    Y     |    Y     | Y        |
+| [扩展统计信息（多列）](/statistics.md)                    | 实验特性 | 实验特性 | 实验特性 | 实验特性 | 实验特性 | 实验特性 | 实验特性 | N        |
+| [统计反馈](/statistics.md#自动更新)                       | 已废弃 | 已废弃 |  已废弃  | 实验特性 | 实验特性 | 实验特性 | 实验特性 | 实验特性 |
+| [快速分析](/system-variables.md#tidb_enable_fast_analyze) | 实验特性 | 实验特性 | 实验特性 | 实验特性 | 实验特性 | 实验特性 | 实验特性 | 实验特性 |
+| [动态剪裁](/partitioned-table.md) | Y | 实验特性 | 实验特性 | 实验特性 | 实验特性 | 实验特性 | N | N |
```
Comment (ran-huang) [inline basic-features.md:131]:
```suggestion
| [动态剪裁](/partitioned-table.md#动态裁剪模式) | Y | 实验特性 | 实验特性 | 实验特性 | 实验特性 | 实验特性 | N | N |
```

**Review by Oreoxmt — APPROVED** [review]

File: basic-features.md [hunk 3]
```diff
@@ -119,61 +121,62 @@ aliases: ['/docs-cn/dev/basic-features/']
...
-| --------------------------------------------------------- | :------: | :------: | :------: | :------: | :------: | :------: | -------- |
-| [CM-Sketch](/statistics.md)                               | 已废弃 |  已废弃  |  已废弃  |  已废弃  |  已废弃  |  已废弃  | Y        |
-| [直方图](/statistics.md)                                  |   Y   |    Y     |    Y     |    Y     |    Y     |    Y     | Y        |
-| [扩展统计信息（多列）](/statistics.md)                    | 实验特性 | 实验特性 | 实验特性 | 实验特性 | 实验特性 | 实验特性 | N        |
-| [统计反馈](/statistics.md#自动更新)                       | 已废弃 |  已废弃  | 实验特性 | 实验特性 | 实验特性 | 实验特性 | 实验特性 |
-| [快速分析](/system-variables.md#tidb_enable_fast_analyze) | 实验特性 | 实验特性 | 实验特性 | 实验特性 | 实验特性 | 实验特性 | 实验特性 |
+| 统计信息                                                  |   6.1    |   6.0    |   5.4    |   5.3    |   5.2    |   5.1    |   5.0    | 4.0      |
+| --------------------------------------------------------- | :------: | :------: | :------: | :------: | :------: | :------: | :------: | -------- |
+| [CM-Sketch](/statistics.md)                               | 已废弃 | 已废弃 |  已废弃  |  已废弃  |  已废弃  |  已废弃  |  已废弃  | Y        |
```
Comment (chrysan) [inline basic-features.md:126]: cmsketch 不是废弃，是在 5.3 以后的版本默认不启用，之前的版本默认还是用的。

File: basic-features.md [hunk 4]
```diff
@@ -119,61 +121,62 @@ aliases: ['/docs-cn/dev/basic-features/']
...
+| [直方图](/statistics.md)                                  |   Y   |   Y   |    Y     |    Y     |    Y     |    Y     |    Y     | Y        |
+| [扩展统计信息（多列）](/statistics.md)                    | 实验特性 | 实验特性 | 实验特性 | 实验特性 | 实验特性 | 实验特性 | 实验特性 | N        |
+| [统计反馈](/statistics.md#自动更新)                       | 已废弃 | 已废弃 |  已废弃  | 实验特性 | 实验特性 | 实验特性 | 实验特性 | 实验特性 |
```
Comment (chrysan) [inline basic-features.md:129]: 这里文字和 Link 不符，应该改成统计信息自动更新

Comment (ran-huang) [inline basic-features.md:129]:
```suggestion
| [统计信息自动更新](/statistics.md#自动更新)                       | 已废弃 | 已废弃 |  已废弃  | 实验特性 | 实验特性 | 实验特性 | 实验特性 | 实验特性 |
```

Comment (ran-huang) [reply basic-features.md]: done

File: basic-features.md [hunk 5]
```diff
@@ -119,61 +121,62 @@ aliases: ['/docs-cn/dev/basic-features/']
...
-| [扩展统计信息（多列）](/statistics.md)                    | 实验特性 | 实验特性 | 实验特性 | 实验特性 | 实验特性 | 实验特性 | N        |
-| [统计反馈](/statistics.md#自动更新)                       | 已废弃 |  已废弃  | 实验特性 | 实验特性 | 实验特性 | 实验特性 | 实验特性 |
-| [快速分析](/system-variables.md#tidb_enable_fast_analyze) | 实验特性 | 实验特性 | 实验特性 | 实验特性 | 实验特性 | 实验特性 | 实验特性 |
+| 统计信息                                                  |   6.1    |   6.0    |   5.4    |   5.3    |   5.2    |   5.1    |   5.0    | 4.0      |
+| --------------------------------------------------------- | :------: | :------: | :------: | :------: | :------: | :------: | :------: | -------- |
+| [CM-Sketch](/statistics.md)                               | 默认关闭 | 默认关闭 |  默认关闭  |  默认关闭  |  Y  |  Y  |  Y  | Y        |
+| [直方图](/statistics.md)                                  |   Y   |   Y   |    Y     |    Y     |    Y     |    Y     |    Y     | Y        |
+| [扩展统计信息（多列）](/statistics.md)                    | 实验特性 | 实验特性 | 实验特性 | 实验特性 | 实验特性 | 实验特性 | 实验特性 | N        |
+| [统计信息自动更新](/statistics.md#自动更新)                       | 已废弃 | 已废弃 |  已废弃  | 实验特性 | 实验特性 | 实验特性 | 实验特性 | 实验特性 |
```
Comment (ran-huang) [inline basic-features.md:129]:
```suggestion
| [统计反馈](/statistics.md#自动更新)                       | 已废弃 | 已废弃 |  已废弃  | 实验特性 | 实验特性 | 实验特性 | 实验特性 | 实验特性 |
```

**Pull request merged**